examples/mujoco/mujoco_env.py

```diff
@@ -37,3 +43,46 @@ def make_mujoco_env(task, seed, training_num, test_num, obs_norm):
         test_envs = VectorEnvNormObs(test_envs, update_obs_rms=False)
         test_envs.set_obs_rms(train_envs.get_obs_rms())
     return env, train_envs, test_envs
+
+
+class MujocoEnvObsRmsPersistence(Persistence):
```

Comment:
Yes, this could not only be applied to the MuJoCo example. But it has something to do with the MuJoCo environment implementation, because the envs can be explicitly configured to be wrapped in a `VectorEnvNormObs` which makes use of the statistic that is persisted here. We could consider adding (an extensible list of) persistence handlers to the `BaseVectorEnv` representation and thus handle this directly in `VectorEnvNormObs`. The higher-level persistence can then retrieve these handlers and apply them.

Reply:
It could be. But I don't think we need to address this in this PR.